signer/src/transaction_coordinator.rs

```diff
@@ -280,12 +281,16 @@ where
         let bitcoin_chain_tip = self
             .context
             .get_storage()
-            .get_bitcoin_canonical_chain_tip()
+            .get_bitcoin_canonical_chain_tip_ref()
             .await?
             .ok_or(Error::NoChainTip)?;
 
         let span = tracing::Span::current();
-        span.record("chain_tip", tracing::field::display(&bitcoin_chain_tip));
+        span.record(
+            "bitcoin_tip_hash",
+            tracing::field::display(bitcoin_chain_tip.block_hash.to_string()),
```

Comment:
You shouldn't need the `to_string()` here, just a reference to the `block_hash`.

Reply:
MIss: [d562ee8](https://github.com/stacks-network/sbtc/pull/1390/commits/d562ee84e2a948ee14c4158749420b1f60e45bb2)